docs/source/release/v6.6.0/Workbench/Bugfixes/33269.rst

```diff
@@ -0,0 +1 @@
+- If the error reporter fails to send, it will now not automatically close
```

Comment:
I would re-word this to say what the reporter does now, maybe this?:
```suggestion
- The error reporter will now stay open and provide an error message if the report failed to send. 
```

Reply:
Much better wording thank you